bootstrap/velocity/src/main/java/org/geysermc/platform/velocity/GeyserVelocityPlugin.java

```diff
@@ -139,6 +138,11 @@ public IGeyserPingPassthrough getGeyserPingPassthrough() {
         return geyserPingPassthrough;
     }
 
+    @Override
+    public File getDataFolder() {
```

Comment:
Same as above

Reply:
Big issue is that bukkit uses file with the same name so I would have to rename the variable entirely. What do you think?

Reply:
Maybe use `getConfigFolder()`?